gmplot/gmplot.py

```diff
@@ -1,22 +1,9 @@
-from __future__ import absolute_import
-
-import json
 import math
-import os
 import requests
-import warnings
-
-from collections import namedtuple
-
-from gmplot.color_dicts import mpl_color_map, html_color_codes
-from gmplot.google_maps_templates import SYMBOLS, CIRCLE
-
-
-Symbol = namedtuple('Symbol', ['symbol', 'lat', 'long', 'size'])
-
```

Comment:
Seems like this PR has many lines that shouldn't be removed or are pulled in from other PRs (e.g. function `get_cycle()`); caused by a rebase onto a different branch?

@brentbiseda, marker titles should already be implemented; if you could figure out what happened with the missing/extra lines here, I can review and merge the label and info box changes.